### PR TITLE
Release of 7.0.0-alpha2

### PR DESCRIPTION
**Pull request by PetrHeinz**
PR created for code review of a new release

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -138,7 +138,7 @@
   },
   "require-dev": {
     "ext-pgsql": "*",
-    "codeception/codeception": "2.4.0",
+    "codeception/codeception": "^2.4.0",
```
Comment (PetrHeinz) [inline composer.json]: Why was it 2.4.0 earlier? How was it resolved?

File: packages/framework/composer.json [hunk 2]
```diff
@@ -50,7 +50,7 @@
         "arvenil/ninja-mutex": "^0.4.1",
         "bmatzner/jquery-bundle": "^2.2.2",
         "bmatzner/jquery-ui-bundle": "^1.10.3",
-        "commerceguys/intl": "0.7.4",
+        "commerceguys/intl": "^0.7.4",
```
Comment (PetrHeinz) [inline packages/framework/composer.json]: Why was it 0.7.4 earlier? How was it resolved?

File: packages/framework/src/Resources/translations/messages.cs.po [hunk 3]
```diff
@@ -1039,6 +1042,9 @@ msgstr "Prvky lze zarovnat pod label"
 msgid "Elements can be ordered behind each other"
 msgstr "Prvky lze dávat za sebe"
 
+msgid "Email"
+msgstr ""
```
[the remark this answers is not in the document]
Comment (MattCzerner) [reply packages/framework/src/Resources/translations/messages.cs.po]: I changed translation string in UserFormType to E-mail

File: composer.json (hunk 1 above)
Comment (MattCzerner) [reply composer.json]: i will add it into commit message after approval of changes

File: packages/framework/composer.json (hunk 2 above)
Comment (MattCzerner) [reply packages/framework/composer.json]: i will add it into commit message after approval of changes

File: composer.json [hunk 4]
```diff
@@ -120,7 +120,7 @@
     "sensio/distribution-bundle": "^5.0.21",
     "sensio/framework-extra-bundle": "^3.0.29",
     "sensio/generator-bundle": "^3.1.7",
-    "snc/redis-bundle": "2.1.0",
+    "snc/redis-bundle": "^2.1.0",
```
[the remark this answers is not in the document]
Comment (MattCzerner) [reply composer.json]: i will add it into commit message after approval of changes

File: CHANGELOG.md [hunk 5]
```diff
@@ -24,6 +24,7 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
+## [7.0.0-alpha2] - 2018-05-24
```
Comment (PetrHeinz) [inline CHANGELOG.md]: I would insert a blank line above so it's clear where to add new change notes.

File: UPGRADE.md [hunk 6]
```diff
@@ -16,7 +16,8 @@ There is a list of all the repositories maintained by monorepo, changes in log b
 * [shopsys/product-feed-interface]
 * [shopsys/product-feed-zbozi]
 
-## [From 7.0.0-alpha1 to Unreleased] 
+## [From 7.0.0-alpha2 to Unreleased]
+## [From 7.0.0-alpha1 to 7.0.0-alpha2]
```
Comment (PetrHeinz) [inline UPGRADE.md]: I would insert a blank line above so it's clear where to add new upgrade notes.

File: packages/framework/src/Resources/translations/messages.cs.po [hunk 7]
```diff
@@ -1891,8 +1888,8 @@ msgstr "Heslo"
 msgid "Password again"
 msgstr "Heslo znovu"
 
-msgid "Password must be at least six characters and can't be the same as login name ."
-msgstr ""
+msgid "Password must be at least six characters and can't be the same as login name."
```
Comment (PetrHeinz) [inline packages/framework/src/Resources/translations/messages.cs.po]: Shouldn't this change be made in the template as well?

File: packages/framework/src/Form/Admin/Customer/UserFormType.php [hunk 8]
```diff
@@ -132,7 +132,7 @@ public function buildForm(FormBuilderInterface $builder, array $options)
                     new Email(['message' => 'Please enter valid e-mail']),
                     new UniqueEmail(['ignoredEmail' => $user !== null ? $user->getEmail() : null]),
                 ],
-                'label' => t('Email'),
+                'label' => t('E-mail'),
```
Comment (PetrHeinz) [inline packages/framework/src/Form/Admin/Customer/UserFormType.php]: A new translation dump should be made after this change (removing the "Email" message)

File: CHANGELOG.md [hunk 9]
```diff
@@ -67,6 +67,7 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
     - all exception interfaces are now Throwable 
     - visibility of all private properties and methods of facades was changed to protected 
         - protected visibility allows overriding of behavior from projects
+- [#130 - License condition for turnover changed from 12 to 3 months](https://github.com/shopsys/shopsys/pull/130)
```
Comment (PetrHeinz) [inline CHANGELOG.md]: Let's not forget force pushing the fixed-up commit into the `license-turnover-period`

File: packages/framework/src/Resources/translations/messages.cs.po [hunk 10]
```diff
@@ -1888,8 +1888,8 @@ msgstr "Heslo"
 msgid "Password again"
 msgstr "Heslo znovu"
 
-msgid "Password must be at least six characters and can't be the same as login name."
-msgstr "Heslo musí obsahovat alespon šest znaků a nemůže být stejné jako přihlašovací jméno."
+msgid "Password must be at least six characters and can't be the same as login name ."
+msgstr "Heslo musí obsahovat alespon šest znaků a nemůže být stejné jako přihlašovací jméno ."
```
Comment (PetrHeinz) [inline packages/framework/src/Resources/translations/messages.cs.po]: Wait what?

File: project-base/composer.json [hunk 11]
```diff
@@ -109,11 +109,11 @@
     "require-dev": {
         "ext-pgsql": "*",
         "ext-zip": "*",
-        "codeception/codeception": "2.4.0",
+        "codeception/codeception": "^2.4.0",
```
Comment (PetrHeinz) [inline project-base/composer.json]: There were 3 changes in the monorepo's composer.json but only 1 change outside of it. Seems that you missed something...

**Comment by PetrHeinz** [comment]
Merged manually in 20b0a25ba.

**Pull request closed without merging**